component/component.js

```diff
@@ -49,8 +49,8 @@ export default Ember.Component.extend(NodeDriver, {
     let config = get(this, 'globalStore').createRecord({
       type: '%%DRIVERNAME%%Config',
       instanceType: 'g6-standard-4', // 4 GB Ram
-      region: 'us-east', // Newark
-      image: 'linode/ubuntu18.04', // 10 year support from Ubuntu
+      region: 'us-iad', // Newark
```

Comment:
```suggestion
      region: 'us-mia', // Miami
```
Match region with comment and PR description.